.github/pull-request-template.md

```diff
@@ -4,9 +4,16 @@
 
 #### Description
 Describe your changes here.
+
+#### Release note
+REQUIRED: Enter a simple message for this commit which will go in the release notes. Include how this will affect a project or a developer.
```

Comment:
Perhaps this should be prefixed with Bugfix or Feature to easily sort in the release notes.

Reply:
Good idea.